yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -542,13 +539,13 @@ class FlatMapOperator : public FlowableOperator<T, R, FlatMapOperator<T, R>> {
     void drainImpl() {
       // phase 1: clear out terminated subscribers
       {
-        auto clearList = [](auto& list) {
+        SubscriberList trash;
+        auto clearList = [&trash](auto& list) {
```

Comment:
nit: if you make the 'trash' another input to the function, then you might declare that variable when the lambda gets used.

Reply:
I'm not sure I follow, you mean don't capture trash, but pass it in as another parameter?

Reply:
nit: would it be better not not?

Reply:
I think it won't matter; in any optimized build the lambda will be inlined